app/src/main/res/values-tr/strings.xml

```diff
@@ -147,124 +167,145 @@
         <item quantity="one">%d çalma listesi</item>
         <item quantity="other">%d çalma listesi</item>
     </plurals>
-    <plurals name="n_week" tools:ignore="MissingQuantity">
```

Comment:
Here you've deleted the tools:ignore, you should either let the ignore or follow android's standards.